python-sdk/tests/data/sample_without_unicode.csv

```diff
@@ -0,0 +1,3 @@
+id,name
+1,First
+2,Second
```

Comment:
Are there any issues with using our existing examples with Unicode in MSSQL?

Reply:
yes we get garbled text into the database when we try to insert the unicode data. So the tests with unicode data will fail

Reply:
will add a note in the documentation that unicode isnt supported in this implementation